### ISSUE TITLE
examples are outdated

### ISSUE THREAD

**Issue by wowselim** [liferay/senna.js#206]
### Expected behaviour

I thought that by running the examples in a webserver I could get an idea of how senna works.

### Actual behaviour

Several resources, including senna.js are not found and the examples look awful and don't work.

### Steps to reproduce

Run the examples.

### Browsers affected

unrelated to browsers.

**Comment by brunobasto** [comment]
Did you run "gulp server"? It builds stuff needed for the examples to run.

**Comment by wowselim** [comment]
This was my experience with README.md:

![image](https://cloud.githubusercontent.com/assets/5699586/24081222/f1be57ee-0caf-11e7-8573-34aa34eca627.png)

Apparently this doesn't work well with windows. I don't even understand why I have to run all these commands just to get some examples to work in my browser...

**Comment by wowselim** [comment]
After installing bower with the -g flag (not specified in readme) I got following error:
![image](https://cloud.githubusercontent.com/assets/5699586/24081280/25d88a8a-0cb1-11e7-8622-8e01fea1d274.png)

Translation: `%1 is not a valid win32 application`

**Comment by brunobasto** [comment]
Hey @wowselim, I just tested on my windows machine and all I had to do was:

1. Clone the repo
2. run "npm install"
3. start an http-server on the root folder

It's expected to work because we push the built files to github. So you don´t really need to build anything unless you want to change stuff. So, functionality wise, everything should work. Although we do depend on some bower dependencies for CSS on the examples.

![Windows 10](http://imgur.com/download/ZluKnPm "Windows 10")
